homeassistant/components/demo/lock.py

```diff
@@ -33,6 +33,7 @@ async def async_setup_platform(
             DemoLock("Kitchen Door", STATE_UNLOCKED),
             DemoLock("Poorly Installed Door", STATE_UNLOCKED, False, True),
             DemoLock("Openable Lock", STATE_LOCKED, True),
+            DemoLock("Another Openable Lock", STATE_UNLOCKED, True),
```

Comment:
Are these for testing?
If so, we should move them to the `kitchen_sink` integration, or just create lock entities in tests.

Reply:
Yes, I need at least 2 openable locks for a group test